fintorch/layers/explainable/LeakyReLU.py

```
@@ -0,0 +1,4 @@
+import torch.nn as nn
+
+class LeakyReLU(nn.LeakyReLU):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)